src/casync-tool.c

```diff
@@ -1081,6 +1081,22 @@ static int verbose_print_done_extract(CaSync *s) {
                          format_bytes(buffer, sizeof(buffer), n_remote_bytes),
                          total_bytes > 0 ? n_remote_bytes * 100U / total_bytes : 0);
 
+        r = ca_sync_get_seed_time(s, &time_nsec);
+        if (!IN_SET(r, -ENODATA, -ENOTTY)) {
+                if (r < 0)
+                        return log_error_errno(r, "Failed to determine seeding time: %m");
+
+                log_info("Time spent seeding: %" PRIu64 " nsecs", time_nsec);
```

Comment:
I think the SI spelling of the unit is `ns`, let's follow that here too, i.e. nsecs → ns

Reply:
(btw, i wonder if we should steal systemd's format_timespan() for this stuff, it should make things a lot more readable)

Reply:
I imported `format_timespan()`, see below.